NitroxServer/GameLogic/StoryManager.cs

```diff
@@ -44,6 +49,25 @@ public StoryManager(PlayerManager playerManager, PDAStateData pdaStateData, Stor
         
         AuroraCountdownTimeMs = auroraExplosionTime ?? GenerateDeterministicAuroraTime(seed);
         AuroraWarningTimeMs = auroraWarningTime ?? ElapsedMilliseconds;
+        // +27 is from CrashedShipExploder.IsExploded, -480 is from the default time (see TimeKeeper)
+        AuroraRealExplosionTime = auroraRealExplosionTime ?? AuroraCountdownTimeMs * 0.001 + 27 - 480;
+
+        timeKeeper.OnTimeSkipped += timeSkipped =>
+        {
+            // Readjust the aurora real explosion time when time skipping because it's based on in-game time
+            if (AuroraRealExplosionTime > timeKeeper.RealTimeElapsed)
+            {
+                double newTime = timeKeeper.RealTimeElapsed + timeSkipped;
+                if (newTime > AuroraRealExplosionTime)
+                {
+                    AuroraRealExplosionTime = timeKeeper.RealTimeElapsed;
+                }
+                else
+                {
+                    AuroraRealExplosionTime -= timeSkipped;
+                }
+            }
+        };
```

Comment:
Not strictly necessary but good practice to dispose this event handler (implement IDisposable on StoryManager). 
This will make it less likely to introduce bugs if StoryManager will be recreated in future of Nitrox.